lib/CModHandler.cpp

```diff
@@ -663,6 +663,14 @@ void CModHandler::loadConfigFromFile (std::string name)
 	logMod->debug("\tWINNING_HERO_WITH_NO_TROOPS_RETREATS\t%d", static_cast<int>(settings.WINNING_HERO_WITH_NO_TROOPS_RETREATS));
 	settings.BLACK_MARKET_MONTHLY_ARTIFACTS_CHANGE = hardcodedFeatures["BLACK_MARKET_MONTHLY_ARTIFACTS_CHANGE"].Bool();
 	logMod->debug("\tBLACK_MARKET_MONTHLY_ARTIFACTS_CHANGE\t%d", static_cast<int>(settings.BLACK_MARKET_MONTHLY_ARTIFACTS_CHANGE));
+	settings.ATTACK_POINT_DMG_MULTIPLIER = hardcodedFeatures["ATTACK_POINT_DMG_MULTIPLIER"].Float();
+	logMod->debug("\tATTACK_POINT_DMG_MULTIPLIER\t%d", static_cast<float>(settings.ATTACK_POINT_DMG_MULTIPLIER));

```

Comment:
Use %f for floats

Reply:
I cast as I follow another positions in hardcoded features, and did not find support for double so casted it to float, relying on automatic float->double conversion after doing that

Reply:
`static_cast<int>` is here for boolean fields. Doubles should print fine with `%f`

Reply:
Ah yes, I thought you mean 
hardcodedFeatures["ATTACK_POINTS_DMG_MULTIPLIER_CAP"].Float()